src/main/webapp/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Black Student Insight</a></li> 
+          <li><a href="college-info.html">Find a School</a></li>
+        <!-- <li><a href="contact-us.html">Contact Us</a><li>  -->
+          <!-- future contact page? -->
+          </ul>
+        <input type="search" id="school-search" onsearch="loadSchoolInfo()" 
+        placeholder="Find a School" onkeypress="hideElements()"/>
+        <!-- <home-button onsearch="loadSchoolInfo()">Search</home-button> -->
+      </nav> 
+    </header>
+    
+    <div id= "information">
+      
+      <h2>Our Focus</h2>
+      <br>
+      <p>Black Student Insight serves as a platform for current minority students 
+          to share their experiences and beliefs regarding their campuses, but also 
+          allows prospective students to look at diversity breakdowns, as well as
+          average admissions statistics for their university of interest.
+      </p>
+      <br>
+      <!-- insert diversity-image.jpg
+      <br> -->
+<!-- note: get images from sydney & toba to add to this section of home page
+      <h2>Meet The Creators</h2>
+      <figure>
+        <a href="https://www.linkedin.com/in/jalen-jenkins-b75772186"><img src="jalen" 
+        alt="Jalen Jenkins" width="300" height="300">
+        <figcaption>Jalen is currently a Computer Science student in Cornell
+            University's Class of 2023. He intends to pursue a entrepreneurial endeavors
+             in the future!</figcaption>
+      </figure>
+      <figure>
+        <a href="https://www.linkedin.com/in/sydney-beasley-6b04b8168"><img src="sydney" 
+        alt="Sydney Beasley" width="300" height="300">
+        <figcaption>Sydney is currently a Computer Science student in Xavier
+            University of Louisiana's Class of 2022. She aspires to become a UX Designer 
+            after graduating!</figcaption>
+      </figure>
+      <figure>
+        <a href="https://www.linkedin.com/in/toba-olokungbemi"><img src="toba" 
+        alt="Toba Olokungbemi" width="300" height="300">
+        <figcaption>Toba is currently a Computer Science student in Harvard
+            University's Class of 2022. He intends to become a Software Engineer 
+            after graduating!</figcaption>
+      </figure> -->
```

Comment:
Shouldn't this be displayed on the page?

Reply:
I didn't want to add that without headshots from you & Toba first